README

```diff
@@ -0,0 +1,38 @@
+# token-tests
+
+Test suite for a standard "Maker-style" ERC20 token. An example of such a standard token can be found in `src/tests/SampleToken.sol`.
+
+## Getting Started
+
+Run `forge install makerdao/token-tests` in your newly setup repository.
+
+There are two possible ways to use `token-tests`
+
+### 1) Letting `token-tests` handle testing for you
+
+That's the recommended way. Inherit your test contract from `token-tests/TokenTests.sol` instead of `std-forge/Test.sol`. You will then need to set the following inherited variables:
+
+```
+    function setUp() public {
+        _token_ = address(new SampleToken());
+        _contractName_ = "SampleToken";
+
+        _tokenName_ = "Sample Token"; // this can be omitted, in which case the correctness of the token name won't be tested (not recommended)
```

Comment:
```suggestion
        _token_ = address(new YourToken());
        _contractName_ = "YourToken";

        _tokenName_ = "Your Token"; // this can be omitted, in which case the correctness of the token name won't be tested (not recommended)
```